lib/config.coffee

```diff
@@ -20,6 +20,10 @@ module.exports =
     type: 'boolean'
     default: true
     order: 4
+  rustSupport:
+    type: 'boolean'
+    default: true
+    order: 4
```

Comment:
To me, this is the same as saying that 'rust' should be right after 'python'. The sort algorithm using the order value seems to respect the order of the input.
Otherwise, I'd have to adjust all orders of each item, which seemed unnecessary.
Do you think this can be done that way ?